### PR TITLE
chore: Bi weekly periods (DHIS2-2971)

### PR DESCRIPTION
**Pull request by edoardo**
Fixes DHIS2-2971.

Changes proposed in this pull request:

- implement bi-weekly periods in shared period selector component used in new analytics apps
- added tests for all the supported fixed periods, this required to change `new Date()` to `new Date(Date.now())` to be able to mock Date.now() and get consistent test results over time.

### CODE REVIEW

**Comment by edoardo** [comment]
There are no tests at the moment. I need to add some.

**Comment by edoardo** [comment]
Tests added. PR ready for review.

File: packages/period-selector-dialog/src/modules/__tests__/FixedPeriodsGenerator.spec.js [hunk 1]
```diff
@@ -0,0 +1,507 @@
...
+            expect(periods[0]).toEqual({
+                startDate: '2019-01-01',
+                endDate: '2019-01-31',
+                name: 'January 2019',
+                iso: '201901',
+                id: '201901',
+            });
+        });
+
+        it('should return the correct object for bi-week 26', () => {
```
Comment (jenniferarnesen) [inline packages/period-selector-dialog/src/modules/__tests__/FixedPeriodsGenerator.spec.js]: Fix test title

File: packages/period-selector-dialog/src/modules/__tests__/FixedPeriodsGenerator.spec.js [hunk 2]
```diff
@@ -0,0 +1,507 @@
...
+                id: '201901B',
+            });
+        });
+
+        it('should return the correct object for bi-month 6', () => {
+            expect(periods[5]).toEqual({
+                startDate: '2019-11-01',
+                endDate: '2019-12-31',
+                name: 'November - December 2019',
+                iso: '201911B',
```
Comment (jenniferarnesen) [inline packages/period-selector-dialog/src/modules/__tests__/FixedPeriodsGenerator.spec.js]: Are the iso and id values correct?  I would have thought '201906B'

Comment (edoardo) [reply packages/period-selector-dialog/src/modules/__tests__/FixedPeriodsGenerator.spec.js]: They are correct, cfr documentation and api:
https://docs.dhis2.org/master/en/developer/html/webapi_date_perid_format.html#
https://play.dhis2.org/dev/api/32/periodTypes.json

Comment (edoardo) [reply packages/period-selector-dialog/src/modules/__tests__/FixedPeriodsGenerator.spec.js]: It was indeed wrong. Fixed.

**Pull request merged**